base.h

```diff
@@ -15,4 +15,19 @@ bytes b32_decode(const std::string& in);
 std::vector<uint16_t> b2048_encode(const bytes& in);
 bytes b2048_decode(const std::vector<uint16_t> &in);
 
+template<typename T> 
+bytes number_to_bytes(T val)
```

Comment:
Could you use put opening braces on the previous lines for consistency with the rest?

Reply:
Will do. I missed this when I was going back and reformatting.